tests/unit/createRoute.ts

```diff
@@ -14,48 +14,6 @@ suite('createRoute', () => {
 		});
 	});
 
-	test('exec() can be customized', () => {
```

Comment:
The options are no longer merged into the instance, so the test is no longer possible.